### PR TITLE
Adding script to index install results

### PR DESCRIPTION
**Pull request by jtaleric**
Allow the post-tasks to index install results

### CODE REVIEW

File: OCP-4.X/roles/install-on-aws/tasks/main.yml [hunk 1]
```diff
@@ -194,6 +194,11 @@
     - dest: "{{ansible_user_dir}}/.kube/config"
       become: false
 
+- name: Copy the install log to the root directory
+  copy:
+    src: "{{ansible_user_dir}}/scale-ci-deploy/scale-ci-aws/.openshift_install.log"
+    dest: "/root/openshift-install.log"
```
Comment (chaitanyaenr) [inline OCP-4.X/roles/install-on-aws/tasks/main.yml:200]: Maybe change this to {{ansible_user_dir}}/openshift-install.log?

File: OCP-4.X/roles/post-install/tasks/main.yml [hunk 2]
```diff
@@ -13,6 +13,17 @@
...
+- name: Add Index install timings to Elasticsearch
+  copy:
+    src: index-install-data.sh
+    dest: "/tmp/index-install-data.sh"
+
+- name: Index install log
+  shell: |
+    export ELASTIC_URL={{elastic}}
+    export ELASTIC_USER={{elastic_user}}
+   ./tmp/index-install-data.sh
```
Comment (mohit-sheth) [inline OCP-4.X/roles/post-install/tasks/main.yml:25]: . /tmp/index-install-data.sh, right?

Comment (chaitanyaenr) [inline OCP-4.X/roles/post-install/tasks/main.yml:25]: s/"./tmp/index-install-data.sh"/"/tmp/index-install-data.sh"

Comment (jtaleric) [reply OCP-4.X/roles/post-install/tasks/main.yml:25]: Good catch!

**Comment by mohit-sheth** [comment]
LGTM

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**